### ISSUE TITLE
Qtile loads for about half a second, then crashes.

### ISSUE THREAD

**Issue by amogusussy** [qtile/qtile#5140]
### Issue description

When I run qtile with `dbus-launch --sh-syntax --exit-with-session qtile start`, qtile loads for a short amount of time (usually 0.5-1 second), then crashes. There is no extra information given by the Xorg exiting message, and other window managers (e.g. xfwm4) work fine, so I doubt it's an issue with Xorg.

The `~/.local/share/qtile/qtile.log` file simply talks about an unknown keysym `10`, but when I look through my key definitions, I can't see any issues.

Also, thank you for all the work that has been done on Qtile. I've been using it for about a year, and it's by far my favorite window manager I've used.

### Version

0.29.0

### Backend

X11 (default)

### Config

config.py:
```python
from keys import keys, mouse
from libqtile import layout, hook, bar, widget
from groupbox_2 import GroupBox2
import global_variables as G
import subprocess
import theme

BORDER_WIDTH = 2
MARGIN_DEFAULT = 4
DESKTOPS = 10
BAR_SIZE = 24
COLORS = theme.Theme()
BAR_PAD = widget.TextBox()

BAR = lambda: bar.Bar(
    [
        widget.TextBox(
            " ◉",
            fontsize=BAR_SIZE - 2,
            mouse_callbacks={
                "Button1": lazy.spawn("rofi -show drun"),
            }
        ),
        GroupBox2(
            normal_style={"text_color": COLORS.workspace_norm},
            has_windows_style={"text_color": COLORS.workspace_active},
            active_any_screen_style={"line": 1},
            disable_drag=True,
            fontsize=17,
        ),
        widget.Spacer(),
        widget.Clock(format=f"󱑆 %H:%M |  %G %-e %b"),
        widget.Spacer(),
        widget.CPU(
            format="CPU {load_percent}%",
        ),
        BAR_PAD,
        widget.Memory(
            format=" {MemUsed:.01f} GB",
            measure_mem="G",
        ),
        widget.TextBox(),
    ],
    BAR_SIZE,
    border_color=COLORS.border,
    border_width=[0] * 4,
    background=COLORS.bar_bg,
    foreground=COLORS.bar_fg,
    margin=[2, 4, 0, 4]
)

screens = [
    Screen(
        wallpaper=COLORS.wallpaper_file,
        wallpaper_mode='stretch',
        top=BAR(),
    ),
    Screen(
        wallpaper=COLORS.wallpaper_file,
        wallpaper_mode='stretch',
        top=BAR(),
    )
]

# Run ~/.config/qtile/autostart.sh on startup
# hook.subscribe.startup_once(lambda: subprocess.run(
#     G.HOME_DIR +  "/.config/qtile/autostart.sh"
# ))

@lazy.function
def minimize_all(qtile):
    for win in qtile.current_group.windows:
        if hasattr(win, "toggle_minimize"):
            win.toggle_minimize()

# keys += [
#     Key([G.MOD], "n", minimize_all(), desc="Toggle minimization on all window"),
# ]

groups = [Group(str(i)) for i in range(1, DESKTOPS + 1)]
for i in groups:
    keys.extend([
        Key([G.MOD], i.name, lazy.group[i.name].toscreen()),
        Key([G.MOD, "shift"], i.name, lazy.window.togroup(i.name))
    ])

layouts = [layout.Columns(
    border_focus=COLORS.focused,
    border_normal=COLORS.unfocused,
    border_width=BORDER_WIDTH,
    border_on_single=True,
    margin=[MARGIN_DEFAULT] * 4
)]

widget_defaults = {
    "font": "JetBrains Mono",
    "fontsize": BAR_SIZE // 2,
    "padding": 3,
    "foreground": COLORS.bar_fg
}
extension_defaults = widget_defaults.copy()

floating_layout = layout.Floating(
    border_focus=COLORS.focused,
    border_normal=COLORS.unfocused,
    border_width=BORDER_WIDTH,
    float_rules=[
        Match(wm_class='V Picture-in-Picture'),
    ]
)

# Fix broken Java applications
wmname = "LG3D"
```

Keys.py:

```python
from libqtile.config import Key  # Import Key, not EzKey
from libqtile.lazy import lazy
from libqtile.config import Click, Drag
import global_variables as G

alacritty_bin = f"alacritty --config-file {G.HOME_DIR}/.config/alacritty/{G.THEME_NAME}.toml"
rofi_bin = f"rofi -show drun -theme-str '@theme \"{G.THEME_NAME}\"'"
G.MOD = "mod4"

keys = [
    # Focus windows. Cycle through both tiling and floating windows.
    Key([G.MOD], "j", lazy.group.next_window(), desc="Cycle to next window"),
    Key([G.MOD], "k", lazy.group.prev_window(), desc="Cycle to previous window"),

    # Move windows
    Key([G.MOD, "shift"], "h", lazy.layout.shuffle_left(), desc="Move window left"),
    Key([G.MOD, "shift"], "l", lazy.layout.shuffle_right(), desc="Move window right"),
    Key([G.MOD, "shift"], "j", lazy.layout.shuffle_down(), desc="Move window down"),
    Key([G.MOD, "shift"], "k", lazy.layout.shuffle_up(), desc="Move window up"),

    # Resize windows
    Key([G.MOD], "h", lazy.layout.grow_left(), desc="Increase window size to the left"),
    Key([G.MOD], "l", lazy.layout.grow_right(), desc="Increase window size to the right"),
    Key([G.MOD, "control"], "j", lazy.layout.grow_down(), desc="Increase window size downward"),
    Key([G.MOD, "control"], "k", lazy.layout.grow_up(), desc="Increase window size upward"),
    Key([G.MOD, "shift"], "n", lazy.layout.normalize(), desc="Return all windows to default width"),

    # Misc
    Key([G.MOD], "q", lazy.window.kill(), desc="Kill active window"),
    Key([G.MOD], "f", lazy.window.toggle_fullscreen(), desc="Enter fullscreen"),
    Key([G.MOD, "shift"], "space", lazy.window.toggle_floating(), desc="Toggle floating window"),
    Key([G.MOD, "control"], "r", lazy.reload_config(), desc="Reload config"),
    Key([G.MOD, "control"], "l", lazy.spawn("pkill librewolf"), desc="Close all librewolf instances"),

    # Hide bar
    Key([G.MOD], "b", lazy.hide_show_bar("top"), desc="Toggle bar"),

    # App launch
    Key([G.MOD], "Return", lazy.spawn(alacritty_bin), desc="Open terminal"),
    Key([G.MOD], "e", lazy.spawn(f"{alacritty_bin} -e env EDITOR=nvim ranger"), desc="Open file manager"),
    Key([G.MOD], "space", lazy.spawn("rofi -show drun"), desc="Launch rofi"),
    Key([G.MOD], "a", lazy.spawn(f"{alacritty_bin} -e pulsemixer"), desc="Open pulsemixer"),

    Audio
    Key([], "XF86AudioPlay", lazy.spawn("audio play-pause"), desc="Toggle audio stream (play/pause)"),
    Key([], "XF86AudioNext", lazy.spawn("audio next"), desc="Next in playlist"),
    Key([], "XF86AudioPrev", lazy.spawn("audio prev"), desc="Last in playlist"),
]


# Drag floating layouts.
mouse = [
    Drag([G.MOD], "Button1", lazy.window.set_position_floating(),
         start=lazy.window.get_position()),
    Drag([G.MOD], "Button3", lazy.window.set_size_floating(),
         start=lazy.window.get_size()),
    Click([G.MOD], "Button2",
          lazy.window.bring_to_front()),
]
```

global_variables.py:
```python
import os

THEME_NAME = "tokyonight"
MOD = "mod4"
HOME_DIR = os.path.expanduser("~")
```

### Logs

```text
2024-12-27 17:12:52,460 WARNING libqtile utils.py:send_notification():L298 dbus-next is not installed. Unable to send notifications.
2024-12-27 17:12:52,603 ERROR libqtile start.py:start():L111 Qtile crashed
Traceback (most recent call last):
  File "/usr/lib/python3.13/site-packages/libqtile/scripts/start.py", line 109, in start
    q.loop()
    ~~~~~~^^
  File "/usr/lib/python3.13/site-packages/libqtile/core/manager.py", line 204, in loop
    asyncio.run(self.async_loop())
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/asyncio/runners.py", line 194, in run
    return runner.run(main)
           ~~~~~~~~~~^^^^^^
  File "/usr/lib/python3.13/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "/usr/lib/python3.13/asyncio/base_events.py", line 720, in run_until_complete
    return future.result()
           ~~~~~~~~~~~~~^^
  File "/usr/lib/python3.13/site-packages/libqtile/core/manager.py", line 216, in async_loop
    self.load_config(initial=True)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/site-packages/libqtile/core/manager.py", line 145, in load_config
    self.grab_key(key)
    ~~~~~~~~~~~~~^^^^^
  File "/usr/lib/python3.13/site-packages/libqtile/core/manager.py", line 502, in grab_key
    syms = self.core.grab_key(key)
  File "/usr/lib/python3.13/site-packages/libqtile/backend/x11/core.py", line 498, in grab_key
    keysym, modmask = self.lookup_key(key)
                      ~~~~~~~~~~~~~~~^^^^^
  File "/usr/lib/python3.13/site-packages/libqtile/backend/x11/core.py", line 485, in lookup_key
    raise utils.QtileError(f"Unknown keysym: {key.key}")
libqtile.utils.QtileError: Unknown keysym: 10
```

### Required

- [X] I have searched past issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by elParaguayo** [comment]
That error probably is the problem for you. However, I don't see which binding is causing that.

Is that `Keys.py` file a straight paste? If so, you've got an error here:
```python
    Audio
    Key([], "XF86AudioPlay", lazy.spawn("audio play-pause"), desc="Toggle audio stream (play/pause)"),
```
`Audio` should be commented out.

**Comment by amogusussy** [comment]
Sorry, it wasn't a direct paste. When pasting, I had 
```
    # Audio
    # Key([], "XF86AudioPlay", lazy.spawn("audio play-pause"), desc="Toggle audio stream (play/pause)"),
    # Key([], "XF86AudioNext", lazy.spawn("audio next"), desc="Next in playlist"),
    # Key([], "XF86AudioPrev", lazy.spawn("audio prev"), desc="Last in playlist"),
```
all commented, and then I un-commented it in the github text box, and also uncommented #Audio by accident.

**Comment by amogusussy** [comment]
I've just figured out what the issue is. In line 12, I've got 10 desktops configured, and later in the config, I loop to make a key for each of these desktops in the form <Mod+num>. But, obviously there is no number 10 key on a keyboard, so it's looking for a non-existent key. I must have changed that part of my config yesterday, and forgot to check if it would actually work.
I apologize for any wasted time.
